rag/advanced_rag/knowlege_compile/wiki_incremental.py

```
@@ -0,0 +1,2428 @@
+
+        # LLM confirm for maybe pairs (entity only, 0.75-0.90)
+        if maybe_pairs and chat_mdl:
+            confirmed = await _wiki_confirm_batch(
+                [(e["name"], cname) for e, cname in maybe_pairs],
+                chat_mdl,
+            )
+            confirmed_set = set()
+            for raw_name, cname in confirmed:
+                name_resolution[raw_name] = cname
+                confirmed_set.add(raw_name)
+            for e, cname in maybe_pairs:
```

Ruff (0.16.0)

[warning] Loop control variable `cname` not used within loop body

(B007)